todoman/ui.py

```diff
@@ -251,8 +251,23 @@ def detailed(self, todo):
         return rv
 
     def format_date(self, date):
+        """
+        Returns date in the following format:
+        * if date == today or tomorrow: "Today" or "Tomorrow"
+        * else: return a string representing that date
+        """
         if date:
-            rv = date.strftime(self.date_format)
+            date_tomorrow = datetime.today().date() + timedelta(days = 1)
+            # Get a dictionary which stores the dates for which we don't have to merely
+            # return a date string and map it to the special string we need to return
+            dates_for_which_not_to_return_simple_date = {
+                datetime.today().date() : "Today",
+                date_tomorrow : "Tomorrow" ,
+            }
+            if date in dates_for_which_not_to_return_simple_date.keys():
```

Comment:
`.keys()` is not necessary, just `if date in dates_for_which_not_to_return_simple_date`.